serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc

```diff
@@ -0,0 +1,18 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-functions-accessing-secrets-configmaps"]
+= Accessing secrets and ConfigMaps from Serverless functions
+include::modules/common-attributes.adoc[]
+:context: serverless-functions-secrets
+
+toc::[]
+
+Your functions, once deployed to the cluster, can access data stored in secrets and config maps. This data can be mounted as volumes, or assigned to environment variables. You can configure this access manually by editing the function configuration file or interactively by using the Knative CLI `kn func` commands.
+
+[NOTE]
+====
+To access secrets and config maps, the function needs to be deployed on the cluster. This functionality is not available to a function running locally.
+====
+
+include::modules/serverless-functions-secrets-configmaps-manually.adoc[leveloffset=+1]
```

Comment:
I'd actually put the interactive command to be listed as a first option while the manual as an alternative one.